packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -47,20 +57,36 @@ export class TreeView extends FoundationElement {
         });
     }
 
+    public currentFocused: HTMLElement | TreeItem | null;
+
     private treeItems: HTMLElement[];
 
     public handleBlur = (e: FocusEvent): void => {
         const { relatedTarget, target } = e;
+        if (
+            (relatedTarget === null || relatedTarget instanceof HTMLElement) &&
+            target instanceof HTMLElement &&
+            !this.contains(relatedTarget)
+        ) {
+            const treeView = this as HTMLElement;
```

Comment:
I don't think you need to store `this` in a const - just call `this.setAttribute("tabindex", "0")`